lib/WebSocketInterface.d.ts

```diff
@@ -30,7 +23,7 @@ export class Socket {
 
   onconnect(): void;
 
-  ondisconnect(event: DisconnectEvent): void;
+  ondisconnect(withError: boolean, code?: number, reason?: string): void;
```

Comment:
Following the docs[*], first argument name should be **error**.

[*]: https://jssip.net/documentation/3.8.x/api/socket/#event_ondisconnect